integration-tests/backtest_test.go

```
@@ -179,68 +71,22 @@ func hitEndpoint(route string, method string, payload interface{}, target interf
 func Test_backtestFlow(t *testing.T) {
-	// setup db
-	db, err := util.NewTestDb()
-	require.NoError(t, err)
-	err = cleanupUniverse(db) // redundant but ensures tables are empty
+	manager, err := NewTestDbManager()
 	require.NoError(t, err)
 
-	tx, err := db.Begin()
-	require.NoError(t, err)
-	defer tx.Rollback()
+	defer manager.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `manager.Close` is not checked

(errcheck)